src/platform/mod.rs

```diff
@@ -11,9 +12,14 @@ use crate::{config, device, logger};
 
 /// Export the current platform.
 /// For now, only QEMU's Virt board is supported
+#[cfg(not(feature = "platform_visionfive2"))]
 pub type Plat = virt::VirtPlatform;
 
+#[cfg(feature = "platform_visionfive2")]
+pub type Plat = visionfive2::VisionFive2Platform;
```

Comment:
The `platform_virt` feature is never used, we can remove it and assume it is the default.